stdlib/3/types.pyi

```diff
@@ -3,9 +3,14 @@
 
 # TODO parts of this should be conditional on version
 
-from typing import Any, Callable, Dict, Iterator, Optional, Tuple, TypeVar, Union, overload
+from typing import (
+    Any, Callable, Dict, Generic, Iterator, Mapping, Optional, Tuple, TypeVar,
+    Union, overload
```

Comment:
Do you need all of these? I think you may be able to trim this list down to only the ones that this module actually uses.